compressor/conf.py

```diff
@@ -73,13 +73,10 @@ class CompressorConf(AppConf):
 
     # Returns the Jinja2 environment to use in offline compression.
     def JINJA2_GET_ENVIRONMENT():
-        alias = 'Jinja2'
+        alias = 'jinja2'
         try:
-            from django.template.loader import _engine_list
-            engines = _engine_list(alias)
-            if engines:
-                engine = engines[0]
-                return engine.env
+            from django.template import engines
+            return engines[alias].env
```

Comment:
I'm all for that change, but it means the exception we catch below isn't quite right. It should be `IndexError`, right ? Do you think you can try adding a test for that ? Something that overrides settings to omit jinja2 template configuration and then tries to access `COMPRESS_JINJA2_GET_ENVIRONMENT` ?